File: WEB/smartstart_test/src/AnnonceBundle/Entity/Feedbackc.php
<?php

namespace AnnonceBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Feedbackc
 *
 * @ORM\Table(name="feedbackc", indexes={@ORM\Index(name="id_freelancer", columns={"id_freelancer"}), @ORM\Index(name="id_Client", columns={"id_Client"})})
 * @ORM\Entity
 */
class Feedbackc
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="contenu", type="string", length=150, nullable=false)
     */
    private $contenu;

    /**
     * @var \User
     *
     * @ORM\ManyToOne(targetEntity="User")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_Client", referencedColumnName="id")
     * })
     */
    private $idClient;

    /**
     * @var \User
     *
     * @ORM\ManyToOne(targetEntity="User")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_freelancer", referencedColumnName="id")
     * })
     */
    private $idFreelancer;

    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param int $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     * @return string
     */
    public function getContenu()
    {
        return $this->contenu;
    }

    /**
     * @param string $contenu
     */
    public function setContenu($contenu)
    {
        $this->contenu = $contenu;
    }

    /**
     * @return \Client
     */
    public function getIdClient()
    {
        return $this->idClient;
    }

    /**
     * @param \Client $idClient
     */
    public function setIdClient($idClient)
    {
        $this->idClient = $idClient;
    }

    /**
     * @return \Freelancer
     */
    public function getIdFreelancer()
    {
        return $this->idFreelancer;
    }

    /**
     * @param \Freelancer $idFreelancer
     */
    public function setIdFreelancer($idFreelancer)
    {
        $this->idFreelancer = $idFreelancer;
    }



}

File: MOBILE/MOBILE INDIV/CodenameOneConsommationWebService 3/src/Entity/Feedbackc.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entity;

/**
 *
 * @author Yasser Bel Haj Ali
 */
public class Feedbackc {
    int id;
    String Contenu ;
    User idfreelancer ; 
    User idClient ;
    
    
    
      public Feedbackc() {
    }
    

    public Feedbackc(String Contenu) {
        this.Contenu = Contenu;
    }

    public Feedbackc(int id, String Contenu) {
        this.id = id;
        this.Contenu = Contenu;
    }

    public Feedbackc(String Contenu, User idfreelancer) {
        this.Contenu = Contenu;
        this.idfreelancer = idfreelancer;
        
    }

    public Feedbackc(int id, String Contenu, User idfreelancer, User idClient) {
        this.id = id;
        this.Contenu = Contenu;
        this.idfreelancer = idfreelancer;
        this.idClient = idClient;
    }
    
    

    public String getContenu() {
        return Contenu;
    }

    public void setContenu(String Contenu) {
        this.Contenu = Contenu;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public User getIdfreelancer() {
        return idfreelancer;
    }

    public void setIdfreelancer(User idfreelancer) {
        this.idfreelancer = idfreelancer;
    }

    public User getIdClient() {
        return idClient;
    }

    public void setIdClient(User idClient) {
        this.idClient = idClient;
    }

  
    
    
}
